lib/presentation/widget/upload_request_inside/upload_request_inside_widget.dart

```diff
@@ -89,6 +91,33 @@ class _UploadRequestInsideWidgetState extends State<UploadRequestInsideWidget> {
           Expanded(child: _buildUploadRequestList())
         ],
       ),
+      bottomNavigationBar: StoreConnector<AppState, AppState>(
+          converter: (store) => store.state,
+          builder: (context, appState) {
+            return SearchBottomBarBuilder()
+              .key(Key('search_bottom_bar'))
+              .onSearchActionClick(() {})
+              .build();
+          }),
+      floatingActionButton: StoreConnector<AppState, UploadRequestInsideState>(
+          converter: (store) => store.state.uploadRequestInsideState,
+          builder: (context, urState) {
+            return (urState.uploadRequestGroup?.collective == false &&
+                  urState.uploadRequestDocumentType == UploadRequestDocumentType.root)
+              ? FloatingActionButton(
```

Comment:
We should not add the button (view) which is not implement logic. Each PR only to focus to the scope of its ticket.

Reply:
I think it should be displayed as the design. Some views also haven't been implemented yet: Search button, Item's context menu button... but they are still displayed

Reply:
No, design is not show all the progress of the product, we need to deliver the functions to the user in sprint. But for sure, we will not deliver the function without work -> it will be bad feedback.

> Some views also haven't been implemented yet: Search button, Item's context menu button... but they are still displayed

I know, so that I need to audit it again, next sprint will be a polish sprint. We drop the quality of this project in a long time.

Reply:
Ok, I will remove this